conf/application.conf

```diff
@@ -337,4 +351,10 @@ Test {
 }
 
 Prod {
+  deleteUnusedApplicationsJob {
+    initialDelay = 10m
+    interval = 1d
+    # Should not run in Prod environment
```

Comment:
This is not actually the config for the prod environment. This would have to be set to false in app-config-production